O/OpenCL_Headers/build_tarballs.jl

```diff
@@ -3,33 +3,52 @@
 using BinaryBuilder
 
 name = "OpenCL_Headers"
-version = v"2020.03.13"
+version = v"2022.9.23"
 
-# Collection of sources required to build this package
+# Collection of sources required to complete build
 sources = [
-    ArchiveSource("https://github.com/KhronosGroup/OpenCL-Headers/archive/v2020.03.13.tar.gz",
-                  "664bbe587e5a0a00aac267f645b7c413586e7bc56dca9ff3b00037050d06f476"),
+    GitSource("https://github.com/KhronosGroup/OpenCL-Headers.git", "4c50fabe3774bad4bdda9c1ca92c82574109a74a"),
+    FileSource("https://patch-diff.githubusercontent.com/raw/KhronosGroup/OpenCL-Headers/pull/209.patch",
+               "c3afd4ad0a37f0b61c0b8656ca4914002ba7994bba05aa2c47fde59b652289c9")
```

Comment:
This patch is only for tests, why do we need that?

Reply:
If you try to build the headers following their instructions you will get an error without the patch. I know that the patch is changing a test file, but for some reason they included that in the install target. Should I remove the patch? I am trying to be the most conservative here, but can change it to something else if you prefer.

Reply:
```suggestion
```